### ISSUE TITLE
Not sure how to use this crate (missing documentation?)

### ISSUE THREAD

**Issue by kwinz** [rustyhorde/vergen#256]
Hi!

1. According to the `README.md` there is supposed to be sample code on [docs.rs](https://docs.rs/crate/vergen/latest) but I can't find any sample code there.

The document recursively links to itself:
> See the documentation at [docs.rs](https://docs.rs/vergen/latest/vergen/) for example usage.

I also looked for a sample code directory in the the source code, but didn't immediatly find any.

3. I figured out roughly what the feature flags mean by reading the source code. But no feature flag documentation anywhere else?

4. And some variables are documented in the old v7 version, but are missing in the current documentation https://github.com/rustyhorde/vergen/tree/legacy/v7

5. See also #220

In summary I just spent 15min trying to figure out what to put in my `build.rs` but I still haven't figured it out. Not a very good onboarding experience for new crate users.

**Comment by kwinz** [comment]
Ah ok I found an older version on docs.rs that has documenation https://docs.rs/vergen/8.2.4/vergen/index.html

I was still using 
```
extern crate vergen;
use vergen::{vergen, Config};

fn main(){
...
vergen(SHORT_SHA | COMMIT_DATE).unwrap();
}
```
 sample code that I found on stackoverflow instead of the newer

```
use std::error::Error;
use vergen::EmitBuilder;

fn main() -> Result<(), Box<dyn Error>> {
    // Emit the instructions
    EmitBuilder::builder().emit()?;
    Ok(())
}
```

**Comment by CraZySacX** [comment]
Yep didn't realize they were broken.  There is a PR running to get that fixed.  I'll publish 8.2.6, which should fix that problem.
